rc/bazel.kak

```diff
@@ -0,0 +1,35 @@
+# https://docs.bazel.build/versions/master/skylark/build-style.html
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾ 
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+BUILD %{
```

Comment:
What file is Bazel using? This regex would also match ArchLinux's `PKGBUILD` files, which are shell scripts.

Reply:
They're called `BUILD`, I wasn't sure how to deal with it here. Let's see if it is fixed now.